src/pseudopeople/data/fake_names.py

```diff
@@ -0,0 +1,195 @@
+"""
+This module includes lists of fake first and last names copied from a .pdf
+version of a NORC report on PVS.
+TODO: Find a link to the NORC report Abie used.
+"""
+
+
+_fake_first_name_string = """GIRL
```

Comment:
Why are these multi-line strings? Wouldn't it make more sense for them to be lists?